frontend/src/components/layout/BaseLayout.vue

```diff
@@ -0,0 +1,20 @@
+<template>
```

Comment:
I looked at this component usages, but I'm not sure that I got it's purpose: is it just for that top part with breadcrumbs?

Reply:
Basically yes. My intent was to create a generic layout component, that we can use in all router views :slightly_smiling_face: